picard/ui/theme.py

```diff
@@ -159,18 +222,29 @@ def update_palette(self, palette, dark_theme, accent_color):
     class MacTheme(BaseTheme):
         @property
         def is_dark_theme(self):
+            dark_theme = False
             if not AppKit:
-                return False
-
-            appearance = AppKit.NSAppearance.currentAppearance()
-            try:
-                basic_appearance = appearance.bestMatchFromAppearancesWithNames_([
-                    AppKit.NSAppearanceNameAqua,
-                    AppKit.NSAppearanceNameDarkAqua
-                ])
-                return basic_appearance == AppKit.NSAppearanceNameDarkAqua
-            except AttributeError:
-                return False
+                return dark_theme
+            if self._loaded_config_theme is not None and self._loaded_config_theme != UiTheme.DEFAULT:
```

Comment:
Perhaps initialize to `UiTheme.DEFAULT` in `BaseTheme.__init__()` and simplify  to:

```python
if self._loaded_config_theme != UiTheme.DEFAULT:
```

Reply:
I vaguely remember choosing None for a reason, probably some weird behavior on the Mac while debugging (screenshots), but seems to work fine with the default now.